server/controllers/stock/reports/rumer.report.handlebars

```diff
@@ -77,6 +77,19 @@
               {{/if}}
             {{/each}}
           </tbody>
+
+          <tfoot>
+            <tr>
+              <td colspan="{{headers.length}}"></td>
+              <td></td>
+              <th>{{translate "REPORT.RUMER.TOTAL_DAYS_OF_STOCK_OUT"}}</th>
```

Comment:
I think these should be 'td' not 'th' since they are not in the table header row.

I suggest merging the cell for TOTAL_DAYS_OF_STOCK_OUT with the previous empty cell (with ncols=2 class="text-right") to right-justifiy.